.github/workflows/image.yaml

```diff
@@ -25,6 +25,7 @@ on:
       - release-*
   push:
     branches:
+      - "pull-request/[0-9]+"
```

Comment:
Can we align the entire `on` block with what we have in other repos? For example https://github.com/NVIDIA/gpu-operator/blob/ee9210bbb64a1c6a01ac67f2d592210c6934a09f/.github/workflows/ci.yaml#L17-L22.

Reply:
I don't have a strong opinion here, but I lean towards using a config that already works in other projects. I want to make sure the workflow status from the `pull-request/[0-9]*` branches is reported back to the original PRs.